### PR TITLE
Add virtual env instructions to local installation

### PR DESCRIPTION
**Pull request by astrojuanlu**
Prompted by a conversation with @humitos

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Rendered version: https://docs--8488.org.readthedocs.build/en/8488/development/docs.html#contributing-from-your-local-machine

File: docs/development/docs.rst [hunk 1]
```diff
@@ -22,33 +22,37 @@ you can verify those changes through the documentation generated when you open a
...
+
+#. create a virtual environment with Python 3.6 and activate it:
+
+   .. code-block:: bash
 
-      git clone --recurse-submodules https://github.com/readthedocs/readthedocs.org/
+      $ cd readthedocs.org
+      $ python3.6 -m venv .venv
+      $ source .venv/bin/activate
+      (.venv) $
```
Comment (nienn) [inline docs/development/docs.rst]: I would probably prefer a note here, or even a short explanation, stating that after the `$ source .venv/bin/activate` line the bash would get prefixed with `(.venv)` .
Changing this and the next `.. prompt::` blocks to `.. code-block::` has the disadvantage of the `$` sign now being selectable. Also, it is not immediately clear what is meant by the `(.venv) $` prefix.

Comment (nienn) [reply docs/development/docs.rst]: Aha! Ok, this got solved with recent changes!

Comment (astrojuanlu) [reply docs/development/docs.rst]: Ueah, right!

**Comment by nienn** [comment]
This is much better!
Tested and it seems to work for me.

**Review by humitos — APPROVED** [review]

**Comment by humitos** [comment]
Does make sense to add the virtualenv creation to the "Install common requirements" as well in the installation guide at https://docs.readthedocs.io/en/latest/development/install.html#set-up-your-environment

**Comment by astrojuanlu** [comment]
Thanks @humitos ! Will do right now

**Comment by astrojuanlu** [comment]
Well, on second thought, the virtual environment for https://docs.readthedocs.io/en/latest/development/install.html#set-up-your-environment is not that important, since it's only installing two applications used to orchestrate the rest:

https://github.com/readthedocs/common/blob/5eec3f61b7b0bb1ee39564bdf5d5a6239e8df964/dockerfiles/requirements.txt#L2-L3

Even the Python version is kind of irrelevant.

On the other hand, @nienn opened #8489, so we can continue the discussion about that part of the documentation there.

**Pull request merged**